src/Wizard.js

```diff
@@ -176,6 +176,18 @@ export default class Wizard extends Webform {
       [`${this.wizardKey}-link`]: 'multiple',
     });
 
+    for (const button in this.form.components[this.page].buttonSettings) {
```

Comment:
Why not just add statements to the template that will conditionally add these elements. This seems very inefficient that they are being added to the DOM only to be immediately taken away. I would prefer to just add conditions to the templates themselves.